src/LoopDriver.php

```diff
@@ -7,20 +7,29 @@
     /**
      * Start the event loop.
      *
+     * The loop MUST continue to run until it is either stopped explicitly, no referenced watchers exist anymore, or an
+     * exception is thrown that cannot be handled. Exceptions that cannot be handled are exceptions thrown from an
+     * error handler or exceptions that would be passed to an error handler but non exists to handle them.
+     *
      * @return void
      */
     public function run();
 
     /**
      * Stop the event loop.
      *
+     * When an event loop is stopped, it continues with its current tick and exits the loop afterwards. Multiple calls
+     * to stop MUST be ignored and MUST NOT raise an exception.
+     *
      * @return void
      */
     public function stop();
 
     /**
      * Defer the execution of a callback.
      *
+     * The deferred callable MUST be executed in the next tick of the event loop.
```

Comment:
@WyriHaximus Are you advising against running in next tick specifically or rather against having/using two functions doing nearly the same?

@AndrewCarterUK right, that's why it's documented only on the LoopDriver interface. I'd be against documenting it on the Loop itself. In general, you _shouldn't_ rely on this, but it should be _possible_ to rely on it.